packages/astro/src/core/build/static-build.ts

```diff
@@ -266,6 +331,54 @@ async function cleanSsrOutput(opts: StaticBuildOptions) {
 			})
 		);
 	}
+	try {
+		fs.rmSync(new URL('./package.json', out));
+	} catch (e) {}
+
+	// TODO: fix this!
+	// // Clean out directly if the outDir is outside of root
+	// if (out.toString() !== opts.settings.config.outDir.toString()) {
```

Comment:
left in code?

Reply:
Removed in [0c9ea57](https://github.com/withastro/astro/pull/5297/commits/0c9ea57a8618dd19493da74838c46a2b295c2900)